common/messages.go

```diff
@@ -49,6 +49,7 @@ type SchemaRequestResponse struct {
 	Config         SchemaObject   `json:"config_schema" msgpack:"config_schema"`
 	Request        RequestSchemas `json:"request_schema" msgpack:"request_schema"`
 	RequiredEvents []EventName    `json:"required_events" msgpack:"required_events"`
+	EventHandlers  []EventName    `json:"event_handlers" msgpack:"event_handlers"`
```

Comment:
The `RequiredEvents` was actually originally meant to be what you have with `EventHandlers`. (but a static list instead of being built off of the actually implemented handlers).

I think your solution is better as it avoids the mistake of implementing a  handler and forgetting to add it to `RequiredEvents`.

So I suggest just using one name or the other, but populating it dynamically like what you're doing.

Reply:
Good point lets just go with the original RequiredEvents. Updated to use RequiredEvents array